packages/service-library/src/servicelib/aiohttp/rest_utils.py

```diff
@@ -63,6 +40,11 @@ async def extract_and_validate(request: web.Request):
     Can raise '400 Bad Request': indicates that the server could not understand the request due to invalid syntax
     See https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
     """
+    warnings.warn(
+        "extract_and_validate is deprecated. Use instead servicelib.rest_utils.extract_and_validate",
```

Comment:
could you also put a hint to when this will be removed?

Reply:
next PR :-)